lists2safebrowsing.py

```diff
@@ -269,9 +269,11 @@ def write_safebrowsing_blocklist(domains, output_name, allow_list, log_file,
     output = []
 
     # Add a static test domain to list
+    test_domain = TEST_DOMAIN_TEMPLATE % output_name
+    if version:
+        test_domain = '{0}-{1}'.format(version.replace('.', '-'), test_domain)
```

Comment:
@englehardt, updated the test domain to use `-` only (e.g. `71-0-base-fingerprinting-track-digest256.dummytracker.org`)